### ISSUE TITLE
VoiceOver Accessibility: Thumbs not discoverable in navigation flow

### ISSUE THREAD

**Issue by drager** [Sharcoux/slider#106]
**Describe the bug**
When using VoiceOver on iOS, the slider thumbs cannot be discovered through standard VoiceOver navigation (left/right swipes). While thumbs are accessible when manually touched and can be adjusted, they are not part of the normal VoiceOver navigation flow. Additionally, once a thumb is selected, you cannot swipe left/right to exit back to normal navigation.

**To Reproduce**

Add a basic RangeSlider:

```javascript
<RangeSlider
    minimumValue={0}
    maximumValue={1}
    thumbSize={44}
    style={{
        width: 200,
        height: 100
    }}
/>
```

* Enable VoiceOver on iOS
* Try to navigate to the slider using left/right swipes
* Try manually touching a thumb and then attempt to navigate away using swipes

**Expected behavior**
* Thumbs should be discoverable through standard VoiceOver left/right swipe navigation
* Should be able to navigate to other elements after interacting with a thumb
* Should maintain current direct touch and adjustment functionality

**Screenshots**
N/A

**Please complete the following information:**
 - OS: iOS 16.7.0
 - Browser: iPhone X
 - React Native version: 0.74.5
 - @react-native-assets/slider version: latest

**Additional context**
The Thumb component itself has proper accessibility properties set up (accessibilityRole, accessibilityValue, etc.), but seems to have issues integrating with the VoiceOver navigation flow. We've attempted various workarounds with accessibility props on container Views without success.

The issue might be related to how the `ResponderView` and `Thumb` components interact with the accessibility system, possibly due to the `focusable={false}` and `pointerEvents='box-only'` settings in ResponderView.

**Comment by drager** [comment]
> Could the problem come from React-Native? Are you sure that the issue is from this lib?

Perhaps, but other things in the app that have a11y properties works fine. I think I need to replicate parts what this lib is doing manually to find out.

**Comment by drager** [comment]
@Sharcoux Ok, I found the problem. I could get the issue by recreating some parts of this library's code. The problem is that the container gets a `width` and `height` that is `0` here: https://github.com/Sharcoux/slider/blob/master/src/components/Thumb.tsx#L44-L45 This happens because [thumbRadius](https://github.com/Sharcoux/slider/blob/master/src/components/Thumb.tsx#L75) is `0`. Hardcoding the `width` and `height` to e.g `1` makes it appear in the VoiceOver navigation flow.

I can make a PR, but I'm not really sure how you want it to be in this package. I couldn't see any place where `thumbRadius` is passed to the container nor do I understand why `thumbRadius` determines the `width` and `height` for the container.

**Comment by Sharcoux** [comment]
Thanks a lot for your investigation. I released a fix in version 10.0.0. Please test and update me.

**Comment by drager** [comment]
Yes, now it works! Thank you!
